### PR TITLE
Reference container images by full name

### PR DESCRIPTION
**Pull request by kierendavies**
This is useful for users of Podman, which isn't configured out-of-the-box to use the `docker.io` registry. It's also a good security practice, in case of a misconfigured default registry.

### CODE REVIEW

**Review by chesedo — APPROVED** [review]
Nice, thanks for this @kierendavies

**Comment by kierendavies** [comment]
I don't see a merge button, presumably because of the timeouts in the `platform-test-cargo-shuttle` CI job. Is there anything I can do about that?

**Comment by kierendavies** [comment]
I missed some hard-coded image names. Branch updated. (And also rebased onto latest `main`.)

**Comment by chesedo** [comment]
> I don't see a merge button, presumably because of the timeouts in the `platform-test-cargo-shuttle` CI job. Is there anything I can do about that?

Oh, I was just waiting to get the latest release out before merging all these. Should happen sometime today

**Pull request merged**
